redfish-core/include/utils/hw_isolation.hpp

```diff
@@ -8,6 +10,11 @@ namespace redfish
 namespace hw_isolation_utils
 {
 
+using AssociationsValType =
```

Comment:
Should we put it in line 345, if we are using these only in the scope of that method?

Reply:
Ack